ui-tests/src/test/java/org/openmrs/reference/page/ActiveVisitsPage.java

```diff
@@ -44,6 +45,7 @@ public void search(String text) {
     }
 
     public ClinicianFacingPatientDashboardPage goToPatientDashboardOfLastActiveVisit() {
+    	waiter.until(ExpectedConditions.visibilityOfElementLocated(ACTIVE_PATIENT));
```

Comment:
Fix the indention here

Reply:
There is no identation here, this is the same identation this project had before

Reply:
You may need to cross the existing code indention otherwise `waiter` method is not well indented.